src/Symfony/Component/Mime/Header/IdentificationHeader.php

```diff
@@ -25,9 +25,9 @@ final class IdentificationHeader extends AbstractHeader
     private $idsAsAddresses = [];
 
     /**
-     * @param string|array $ids
+     * @param array|string $ids
```

Comment:
the phpdoc should be changed to `string|string[]` (otherwise, it brings no value). But this change should probably be done in an older branch (and so a separate PR)